internal/pkg/dependencies/repository.go

```diff
@@ -65,7 +66,10 @@ func (c *common) TemplateRepositoryDir(definition model.TemplateRepository, _ mo
 		}
 		return aferofs.NewLocalFs(c.Logger(), path, "")
 	case model.RepositoryTypeGit:
-		panic("support for Git repository is not implemented")
+		if !git.Available() {
+			return nil, fmt.Errorf("git command is not available")
+		}
```

Comment:
Rovno ti to okomentuje,
Toto by som hodil do `CheckoutTemplateRepository` (ktory tiez vracia err).
Asi nie je dovod to mat tu.

Reply:
A chcelo by to nejaku lepsi hlasku, uvidi ju user: `nastroj "git" nenajdeny, ak chcete nacitavat sablony z Git repositara, tak ho prosim nainstalujte`

Reply:
uplne profi by bolo vracat z `CheckoutTemplateRepository`, chybu odchytit podla typu, a v `TemplateRepositoryDir` vratit finalnu hlasku, ... mozeme si to ukazat.